can/interface.py

```diff
@@ -28,19 +28,19 @@
 
 # interface_name => (module, classname)
 BACKENDS = {
-    'kvaser':           ('can.interfaces.kvaser',           'KvaserBus'),
-    'socketcan_ctypes': ('can.interfaces.socketcan',        'SocketcanCtypes_Bus'),
-    'socketcan_native': ('can.interfaces.socketcan',        'SocketcanNative_Bus'),
-    'serial':           ('can.interfaces.serial.serial_can','SerialBus'),
-    'pcan':             ('can.interfaces.pcan',             'PcanBus'),
-    'usb2can':          ('can.interfaces.usb2can',          'Usb2canBus'),
-    'ixxat':            ('can.interfaces.ixxat',            'IXXATBus'),
-    'nican':            ('can.interfaces.nican',            'NicanBus'),
-    'iscan':            ('can.interfaces.iscan',            'IscanBus'),
-    'virtual':          ('can.interfaces.virtual',          'VirtualBus'),
-    'neovi':            ('can.interfaces.ics_neovi',        'NeoViBus'),
-    'vector':           ('can.interfaces.vector',           'VectorBus'),
-    'slcan':            ('can.interfaces.slcan',            'slcanBus')
+    'kvaser':           ('can.interfaces.kvaser',             'KvaserBus'),
+    'socketcan_ctypes': ('can.interfaces.socketcan',          'SocketcanCtypes_Bus'),
+    'socketcan_native': ('can.interfaces.socketcan',          'SocketcanNative_Bus'),
+    'simpleserial':     ('can.interfaces.serial.simpleserial','SimpleSerialBus'),
+    'slcan':            ('can.interfaces.serial.slcan',       'SlcanBus')
```

Comment:
There is a comma missing at the end of this line. I must've missed it while merging some conflict.